app/src/hydrate.ts

```diff
@@ -16,6 +16,7 @@ import {
 	useUserStore,
 	useNotificationsStore,
 } from '@/stores';
+import { useTranslationStrings } from '@/modules/settings/composables/use-translation-strings';
```

Comment:
Seeing that this composable is used outside of the module, lets move this composable to the root composables folders